sklearn/manifold/_mds.py

```diff
@@ -572,6 +573,7 @@ def __init__(
 
     def __sklearn_tags__(self):
         tags = super().__sklearn_tags__()
+        tags.transformer_tags = TransformerTags(preserves_dtype=["float64"])
```

Comment:
Because we don't want to implement `transform` but only `fit_transform` such that you can only provide the same data at `fit` and `transform`.

At least this is what I recall.

Reply:
`TransformerMixin` doesn't do anything to `transform`. Here we'd use its tags, and override `fit_transform` from the mixin.